Gemfile

```diff
@@ -1,8 +1,18 @@
+
 source 'https://rubygems.org'
 
+gem 'pg'
+gem 'sinatra'
+gem 'sinatra-flash'
+gem 'bcrypt'
+
 group :test do
+  gem 'capybara'
   gem 'rspec'
   gem 'rubocop', '0.79.0'
+  gem 'rubocop-rspec'
   gem 'simplecov', require: false
   gem 'simplecov-console', require: false
+  gem 'coveralls', require: false
+  gem 'cucumber'
```

Comment:
According to rubocop, gems should be listed alphabetically within the groups/sections they're in (I learnt this yesterday!).